mzLib/Omics/Digestion/IDigestionParams.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Omics.Fragmentation;
+
+namespace Omics.Digestion
+{
+    public interface IDigestionParams
+    {
+        int MaxMissedCleavages { get; set; }
+        int MinLength { get; set; }
+        int MaxLength { get; set; }
+        int MaxModificationIsoforms { get; set; }
+        int MaxMods { get; set; }
+        DigestionAgent Enzyme { get; }
```

Comment:
might not be an Enzyme. For example cyanogen bromide

Reply:
Renamed Enzyme to DigestionAgent